
The usual label end of ... statement for gcc > 3.4.x.

  - Rene Rebe <rene@rocklinux-consulting.de>

--- alsa-tools-1.0.6/as10k1/as10k1.c	2003-07-15 20:33:59.000000000 +0200
+++ alsa-tools-1.0.6-fixed/as10k1/as10k1.c	2004-09-14 17:11:08.451196236 +0200
@@ -159,6 +159,7 @@
 
 		}
 next_cli_arg:
+		;
 
 	}
 }
